common/app/common/Preload.scala

```diff
@@ -6,11 +6,6 @@ import play.api.mvc.RequestHeader
 
 object Preload {
 
-  def commercialBundleNameAsset(implicit request: RequestHeader): Seq[PreloadAsset] =
```

Comment:
Looks like we used to preload the commercial bundle for Frontend articles and fronts. Since the standalone bundle however this has not been the case. As the switch is now removed we can remove this logic entirely (and maybe revisit preloading).

Reply:
Is this correct @mxdvl wrt when you created the standalone bundle?

Reply:
Yep, that seems to be what I did in https://github.com/guardian/frontend/pull/24252/files#diff-3c1b50075cac36141a4a334f8b871962156699694f082541b18074bd32235ff6